lib/new_relic/agent/configuration/default_source.rb

```diff
@@ -986,22 +1010,25 @@ def self.enforce_fallback(allowed_values: nil, fallback: nil)
           :default => value_of(:disable_memcache_instrumentation),
           :public => true,
           :type => Boolean,
+          :deprecated => true,
           :allowed_from_server => false,
-          :description => 'If <code>true</code>, disables instrumentation for the memcached gem.'
+          :description => deprecated_description(:'instrumentation.memcached', 'If <code>true</code>, disables instrumentation for the memcached gem.')
         },
         :disable_memcache_client => {
           :default => value_of(:disable_memcache_instrumentation),
           :public => true,
           :type => Boolean,
+          :deprecated => true,
           :allowed_from_server => false,
-          :description => 'If <code>true</code>, disables instrumentation for the memcache-client gem.'
+          :description => deprecated_description(:'instrumentation.memcache-client', 'If <code>true</code>, disables instrumentation for the memcache-client gem.')
         },
         :disable_dalli => {
           :default => value_of(:disable_memcache_instrumentation),
           :public => true,
           :type => Boolean,
+          :deprecated => true,
           :allowed_from_server => false,
-          :description => 'If <code>true</code>, disables instrumentation for the dalli gem.'
+          :description => deprecated_description(:'instrumentation.memcache', 'If <code>true</code>, disables instrumentation for the dalli gem.')
```

Comment:
How come this one is set up to just use instrumentation.memcache instead of a new instrumentation.dalli?

Reply:
Design decision.  Nothing was using memcache anyway, so I figured since Dalli is overwhelmingly used, just make memcache drive Dalli.

Reply:
Makes sense to me! Just wanted to make sure we weren't missing something